File: app/src/main/java/com/rex50/mausam/storage/database/key_values/KeyValues.java
package com.rex50.mausam.storage.database.key_values;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "keyValues")
public class KeyValues {

    @PrimaryKey
    @NonNull
    @ColumnInfo(name = "key")
    private String key;

    @NonNull
    @ColumnInfo(name = "value")
    private String value;

    public KeyValues(String key, String value){
        this.key = key;
        this.value = value;
    }

    public String getKey(){
        return this.key;
    }

    public String getValue(){
        return this.value;
    }

}

File: app/src/main/java/com/rex50/mausam/storage/database/key_values/KeyValues.kt
package com.rex50.mausam.storage.database.key_values

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import org.joda.time.DateTime

@Entity(tableName = "keyValues")
class KeyValues(
        @ColumnInfo(name = "key") @PrimaryKey val key: String,
        @ColumnInfo(name = "value") val value: String,
        @ColumnInfo(name = "dateCreated") val dateCreated: DateTime
)